assets/css/base/_base.scss

```diff
@@ -22,3 +24,45 @@ h3 {
 p {
   font-size: 1em;
 }
+
+// Links and Buttons
+a {
+  color: $blue-dark;
+  font-weight: bold;
+  text-decoration: none;
+}
+
+a:hover,
+a:focus {
+  background-color: $orange-focus;
+}
+
+button {
+  background-color: $green-light;
+  border: 2px solid $blue-alt-light;
+  border-radius: 70px;
```

Comment:
It's a good idea to use `ems` here (see following comment). This makes sure the element will scale. You can use the formula (70px / 16px) &times; 1em to figure this out (here that works out to 4.375em).